docs/feature-custom-validators-sanitizers.md

```diff
@@ -62,13 +62,16 @@ app.post('/user', body('email').custom(isValidUser), (req, res) => {
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+> **Note:** In the example above, validation might fail even due to issues with fetching User information. The implications of accessing the data layer during validation should be carefully considered.
+
 ### Example: checking if password confirmation matches password
 
 ```js
 const { body } = require('express-validator');
 
 app.post(
   '/user',
+  body('password').isLength({ min: 5 }),
```

Comment:
This doesn't add that more layer of security, but we can't do anything for that right now.